crates/superposition/Superposition.cac.toml

```diff
@@ -1,9 +1,12 @@
 [default-config]
 mandatory_dimensions = { "value" = [
 ], "schema" = { "type" = "array", "items" = { "type" = "number" } } }
+experiments_webhook_config = { "value" = {}, "schema" = { "type" = "object", properties = {"url" = { "type" = "string" }, "method" = { "enum" = ["Post","Put","Get"], "type" = "string" },"headers" = { "type" = "array", "items" = { "type" = "string", "enum" = ["ConfigVersion"] }}, authorization = {"type" = "object", properties = { "key" = { "type" = "string" }, "value" = { "type" = "string" }}}, "required"= ["url", "method", "headers", "authorization"], "additionalProperties" = false}}}
```

Comment:
you have added this `"required"= ["url", "method", "headers", "authorization"]`, but the default value is set as `"value" = {}`

even though the validation is not enabled, lets add stuff as per proper validations only, 
because I am not sure with all the keys marked as required and then with proper default value, how are we going to disable the webhook call for those who dont need it